redash/query_runner/athena.py

```diff
@@ -40,6 +40,9 @@ def configuration_schema(cls):
             'secret': ['aws_secret_key']
         }
 
+    @classmethod
+    def annotate_query(cls):
+        return False
```

Comment:
I know of several Redash users who rely on this comment for monitoring/auditing purposes. Instead of disabling it, let's add a settings FLAG to decide whether to disable it with the default being `True` (=> enable it).

It can be something like:

```python
from redash.settings import parse_boolean

...

ANNOTATE_QUERY = parse_boolean(os.environ.get('ATHENA_ANNOTATE_QUERY', 'true'))
```

And then use `ANNOTATE_QUERY` as the return value of `annotate_query`.